doc/source/refs.rst

```diff
@@ -21,6 +21,10 @@ References
 .. [BV2004] Boyd, S, and Vandenberghe, L. *Convex optimization*.
    Cambridge university press, 2004.
 
+.. [ComPes2011] Combettes, P.L, and Pesquet, J.-C. *Proximal splitting
```

Comment:
We have to live with this ambiguity, I guess. Next time, you have two authors which start with "Com" and "Pes" which are not the ones we have here. Or, even worse, think of Chinese authors -> unambiguous abbreviation totally impossible. :-)

Go for CP2011c.